account_chatter/models/account_account.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class AccountAccount(models.Model):
+
+    _name = "account.account"
+    _inherit = ['account.account', 'mail.thread']
+    _translate = False
+
+    name = fields.Char(track_visibility=True)
+    currency_id = fields.Many2one(track_visibility=True)
+    code = fields.Char(track_visibility=True)
+    deprecated = fields.Boolean(track_visibility=True)
+    reconcile = fields.Boolean(track_visibility=True)
+    user_type_id = fields.Many2one(track_visibility=True)
+    tax_ids = fields.Many2many(track_visibility=True)
+    tag_ids = fields.Many2many(track_visibility=True)
+    group_id = fields.Many2one(track_visibility=True)
+    realizable_account = fields.Boolean(track_visibility=True)
+
+    def _message_track(self, tracked_fields, initial):
```

Comment:
A docstring would be very welcomed, otherwise one needs to read the whole method to figure out this adds support to track many2many fields.